rllib/agents/dqn/apex.py

```diff
@@ -283,6 +375,260 @@ def add_apex_metrics(result: dict) -> dict:
             merged_op, workers, config, selected_workers=selected_workers
         ).for_each(add_apex_metrics)
 
+    def get_samples_and_store_to_replay_buffers(self):
```

Comment:
Could we add a docstring here and in all new APEXTrainer methods?

Reply:
sounds good to me